node/malus/src/variants/store_malicious_available_data.rs

```diff
@@ -0,0 +1,353 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
Can we avoid duplicating this file: https://github.com/paritytech/polkadot/blob/master/node/core/av-store/src/lib.rs ?